src/commands/data/connectors/update.ts

```diff
@@ -33,12 +33,40 @@ export default class ConnectorsUpdate extends BaseCommand {
         return [key, value]
       },
     }),
+    addTable: flags.string({
+      char: 't',
+      description: 'Tables to add',
+      multiple: true,
+      required: false,
+    }),
+    removeTable: flags.string({
+      char: 't',
+      description: 'Tables to remove',
+      multiple: true,
+      required: false,
+    }),
+    addExclude: flags.string({
```

Comment:
`addExclude` and `removeExclude` are a bit confusing to me. What do you think about: `excludeColumn` and `includeColumn`?

Reply:
I also think addExclude reads awkwardly, i was trying to stay consistent with the flags for create connector which is just `exclude`. 
But if we don't mind that i will change to this cause prefer it as well.

Reply:
Thinking about naming a bit more i think excludeColumn and includeColumn also is awkward. Because you are either addeing another excluded column or removing an already excluded column - not including anoter column.

So I think if anything AddExcludedColumn and RemoveExcludedColumn makes more sense. But thats long. Thoughts?

Reply:
Hmm. I see your point.

I do like `exclude-column`, and would be in favor of changing the flag for `create` to match. I'd vote to use `exclude-column`, because it's shorter and descriptive. We could use the longer `remove-excluded-column` just for removal. (I suspect removing excluded columns will be a pretty rare requirement, so having it be a little long is less of an issue.)

Reply:
I agree that removing an excluded column would be a rare action.

I changed to excludeColumn and removeExcludedColumn. When I use the dashes the parser is unhappy in the flag declaration.